salt/utils/msgpack.py

```diff
@@ -69,12 +69,26 @@ def _sanitize_msgpack_kwargs(kwargs):
     return kwargs
 
 
+def _sanitize_msgpack_unpack_kwargs(kwargs):
+    """
+    Clean up msgpack keyword arguments for unpack operations, based on
+    the version
+    https://github.com/msgpack/msgpack-python/blob/master/ChangeLog.rst
+    """
+    assert isinstance(kwargs, dict)
+    if version >= (1, 0, 0) and kwargs.get("raw", None) is None:
```

Comment:
Why not adding this new condition to the current logic inside `_sanitize_msgpack_kwargs` method?

Reply:
Because is not a valid parameter for the Packager, produces an exception. Another alternative is add a new boolean parameter to `_sanitize_msgpack_kwargs` to indicate is we are packaging or unpackaging, but is equivalent to this one.

Reply:
Thanks for clarifying!